### PR TITLE
Add management sections for staff

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

File: temba/tests/base.py [hunk 1]
```diff
@@ -751,6 +751,19 @@ def assertResponseError(self, response, field, message, status_code=400):
         self.assertTrue(message, isinstance(body[field], (list, tuple)))
         self.assertIn(message, body[field])
 
+    def getContentMenuItem(self, menu, title):
+        for item in menu:
+            if item.get("label", None) == title:
```
Comment (rowanseymour) [inline temba/tests/base.py:756]: we sure play fast and loose with name vs label vs title... my preference is name being an identifier of objects in the workspace, label being a UI text element, title being page title, modal title etc.

File: temba/utils/templatetags/temba.py [hunk 2]
```diff
@@ -79,6 +79,11 @@ def icon(o):
     return ""
 
 
+@register.filter
+def deslug(str):
+    return str.replace("_", " ").title()
```
Comment (rowanseymour) [inline temba/utils/templatetags/temba.py:84]: technically this is dealing with snaked strings.. see `temba.utils.unsnakify` which it should probably call

File: templates/frame.haml [hunk 3]
```diff
@@ -309,7 +308,8 @@
       }
 
       function handlePosterize(ele) {
-        posterize(ele.getAttribute('href'));
+        console.log(ele.getAttribute('href') || ele.dataset.href);
```
Comment (rowanseymour) [inline templates/frame.haml:311]: cruft?

File: templates/includes/nav.haml [hunk 4]
```diff
@@ -183,15 +183,15 @@
         .title
           -trans "channels"
 
-  -if request.user.is_superuser or perms.orgs.user_list
+  -if request.user.is_superuser or request.user.is_staff
```
Comment (rowanseymour) [inline templates/includes/nav.haml:186]: All these can be simplified to `-if request.user.is_staff` because the superuser is staff

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: templates/frame.haml (hunk 3 above)
Comment (ericnewcomer) [reply templates/frame.haml:311]: you and your reviewing of draft PRs

File: temba/tests/base.py (hunk 1 above)
Comment (ericnewcomer) [reply temba/tests/base.py:756]: Don't disagree, but not changing how content menus work in this PR.

File: temba/orgs/tests.py [hunk 5]
```diff
@@ -5689,3 +5703,54 @@ def test_model(self):
...
+
+
+class TestSpa(TembaTest):
+    def test_menu(self):
+
+        # our admin gets a full menu
+        self.login(self.admin)
+        response = self.client.get(reverse("orgs.org_menu"))
+        # TODO: feels like we have our assert params backward and this should be greater
+        self.assertLess(5, len(response.json()["results"]))
```
[the remark this answers is not in the document]
Comment (ericnewcomer) [reply temba/orgs/tests.py:5715]: Which ya, fixing means a pretty big test refactor that is mostly meaningless. 🤷

File: temba/orgs/tests.py [hunk 6]
```diff
@@ -5689,3 +5703,53 @@ def test_model(self):
...
+        self.login(self.customer_support)
+        response = self.client.get(reverse("orgs.org_menu"))
+        self.assertEqual(len(response.json()["results"]), 1)
+
+    def test_staff(self):
+
+        # admin can't see org manage page
+        self.login(self.admin)
+        response = self.client.get(reverse("orgs.org_read", args=[self.org.id]))
+        self.assertRedirect(response, reverse("users.user_login"))
```
Comment (rowanseymour) [inline temba/orgs/tests.py:5726]: don't forget we have a `CRUDLTestMixin.assertStaffOnly`

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**